### PR TITLE
feature: support FTX account information api

### PR DESCRIPTION
**Pull request by ycdesu**
Note that the make/taker fee on FTX will be a very small number, so I modify the bps type from int to float. For example, your maker fee may be 2.5 bps.

### CODE REVIEW

**Review by c9s — APPROVED** [review]

File: pkg/cmd/account.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+		if err != nil {
+			return err
+		}
+
+		session, ok := environ.Session(sessionName)
+		if !ok {
+			return fmt.Errorf("session %s not found", sessionName)
+		}
+
+		b, err := session.Exchange.QueryAccount(ctx)
```
Comment (c9s) [inline pkg/cmd/account.go:64]: b?

File: pkg/cmd/account.go [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+		configFile, err := cmd.Flags().GetString("config")
+		if err != nil {
+			return err
+		}
+
+		if len(configFile) == 0 {
+			return errors.New("--config option is required")
+		}
+
+		if _, err := os.Stat(configFile); os.IsNotExist(err) {
```
Comment (c9s) [inline pkg/cmd/account.go:36]: balances cmd 那邊有改善 error checking ，可以抄那邊的 code

File: pkg/cmd/account.go [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+			return err
+		}
+
+		userConfig, err := bbgo.Load(configFile, false)
+		if err != nil {
+			return err
+		}
+
+		environ := bbgo.NewEnvironment()
+		if err := environ.ConfigureDatabase(ctx); err != nil {
```
Comment (c9s) [inline pkg/cmd/account.go:46]: 這邊應該是可選，用不到 db, 應該可以跳過

File: pkg/exchange/ftx/exchange.go [hunk 4]
```diff
@@ -53,7 +53,7 @@ func (e *Exchange) Name() types.ExchangeName {
 }
 
 func (e *Exchange) PlatformFeeCurrency() string {
-	panic("implement me")
+	return toGlobalCurrency("ftt")
```
Comment (c9s) [inline pkg/exchange/ftx/exchange.go:56]: ftx 內部使用是小寫嗎?

File: pkg/cmd/account.go (hunk 1 above)
Comment (ycdesu) [reply pkg/cmd/account.go:64]: since I copy from another command, XD

**Review by c9s — APPROVED** [review]

**Pull request merged**